tests/utils/dataimportcron.go

```diff
@@ -64,3 +75,18 @@ func NewDataImportCronWithStorageSpec(name, size, schedule, dataSource string, i
 		},
 	}
 }
+
+// VerifyCronJobCleanup verifies the DataImportCron cronjob was deleted
+func VerifyCronJobCleanup(k8sClient *kubernetes.Clientset, cronJobNamespace, cronNamespace, cronName string) {
```

Comment:
I think we are better off not including this debugging utility in the code base; It's probably not that expensive, but, on the logical level we know the cleanup part of the reconcile loop has been there for ages and is covered by unit tests, and we are not aware of any problems with it.
We've identified a scenario where it could get skipped, and the watches should solve that.

Reply:
Removed the func test, added utest instead.